rocketpy/EnvironmentAnalysis.py

```diff
@@ -20,6 +20,17 @@
 from rocketpy.Function import Function
 from rocketpy.units import convert_units
 
+try:
+    import ipywidgets as widgets
+    import jsonpickle
+    from timezonefinder import TimezoneFinder
+    from windrose import WindroseAxes
+except ImportError:
+    raise ImportError(
+        "At least one required module could not be imported."
+        + "The EnvironmentAnalysis requires additional dependencies."
+        + "Please install them by running 'pip install rocketpy[env_analysis]'."
+    )
```

Comment:
Here, I think it would be better to point which module had an import error. I understand that the most common case would be the user not having any of them (since an "common" install was performed), however it certainly helps in debugging if the problematic module is indicated:

```suggestion
try:
    import ipywidgets as widgets
    import jsonpickle
    from timezonefinder import TimezoneFinder
    from windrose import WindroseAxes
except ImportError as error:
    raise ImportError(
        f"The following error was encountered while importing dependencies: '{error}'. "
        "Please note that the EnvironmentAnalysis requires additional dependencies, "
        "which can be installed by running 'pip install rocketpy[env_analysis]'."
    )
```

This is just a suggestion to make it clearer, feel free to change the error message.

Reply:
Great suggestion, many thanks @phmbressan !!